src/MIT.xml

```diff
@@ -9,7 +9,9 @@
          <p>MIT License</p>
       </titleText>
       <copyrightText>
-         <p>Copyright (c) &lt;year&gt; &lt;copyright holders&gt;</p>
+         <p>Copyright (c) 
+		    <alt match=".+" name="copyright">&lt;year&gt; &lt;copyright holders&gt;</alt>
+	     </p>
```

Comment:
Your indent here, and on the previous line, is a strange mix of tabs and spaces.  Since the old `MIT.xml` contained no tabs, you may want to stick with just spaces here.

Reply:
I was matching the style of the BSD license.  Feel free to create a PR against this branch, otherwise it is good enough from my perspective.

Reply:
> Feel free to create a PR against this branch...

Ok, will do.

Reply:
> > Feel free to create a PR against this branch...
>
> Ok, will do.

Missed this branch (sorry for the delay), so I've filed the fixup as #662.